### PR TITLE
Teshari prosthetics

### PR DESCRIPTION
**Pull request by Greenjoe12345**
Adds two sets of Teshari prosthetics
fixes holes in Teshari torso sprites
adds a 'bald' hair for Teshari to use with FBPs
allows Teshari to use FBPs
also adds some canes and a crutch

### CODE REVIEW

File: code/game/objects/items/weapons/cane.dm [hunk 1]
```diff
@@ -53,4 +53,70 @@
...
+
+/obj/item/weapon/cane/white/collapsible
+	name = "telescopic white cane"
+	desc = "A telescopic white cane. They are commonly used by the blind or visually impaired as a mobility tool or as a courtesy to others."
+	icon_state = "whitecane1in"
+	item_state = "whitecanein"
+	item_icons = list(
+			slot_l_hand_str = 'icons/mob/onmob/items/lefthand.dmi',
+			slot_r_hand_str = 'icons/mob/onmob/items/righthand.dmi',
+		)
```
Comment (martinlyra) [inline code/game/objects/items/weapons/cane.dm:88]: Travis is regarding the indentation of list content as closing bracket as "incorrect indentation", 'unindentate' by one tab.

File: code/game/objects/items/weapons/cane.dm [hunk 2]
```diff
@@ -53,4 +53,70 @@
...
+	w_class = ITEM_SIZE_SMALL
+	force = 3
+	var/on = 0
+
+/obj/item/weapon/cane/white/collapsible/attack_self(mob/user as mob)
+	on = !on
+	if(on)
+		user.visible_message("<span class='notice'>\The [user] extends the white cane.</span>",\
+				"<span class='warning'>You extend the white cane.</span>",\
+				"You hear an ominous click.")
```
Comment (martinlyra) [inline code/game/objects/items/weapons/cane.dm:99]: These two lines have double tabs as indentation, and are possibly what cause Travis to go red. Remove one tab from each line to try solve it.

File: code/game/objects/items/weapons/cane.dm [hunk 3]
```diff
@@ -53,4 +53,70 @@
 	else
 		SetName("cane shaft")
 		icon_state = "nullrod"
-		item_state = "foldcane"
\ No newline at end of file
+		item_state = "foldcane"
+
+/obj/item/weapon/cane/crutch
+	name ="crutch"
+	desc = "A long stick with a crosspiece at the top, used to help with walking."
```
Comment (Laniakeaaaa) [inline code/game/objects/items/weapons/cane.dm:60]: icon = 'icons/urist/items/uristweapons.dmi

Repath icons to this, as long as they keep the same icon_state/item state names, they should work fine

File: code/game/objects/items/weapons/cane.dm [hunk 4]
```diff
@@ -53,4 +53,70 @@
...
+	icon_state = "crutch"
+	item_state = "crutch"
+
+/obj/item/weapon/cane/white
+	name = "white cane"
+	desc = "A white cane. They are commonly used by the blind or visually impaired as a mobility tool or as a courtesy to others."
```
Comment (Laniakeaaaa) [inline code/game/objects/items/weapons/cane.dm:66]: Same change here.

File: code/game/objects/items/weapons/cane.dm [hunk 5]
```diff
@@ -53,4 +53,70 @@
...
+		return TRUE
+	else
+		. = ..()
+
+
+//Code for Telescopic White Cane writen by Gozulio
+
+/obj/item/weapon/cane/white/collapsible
+	name = "telescopic white cane"
+	desc = "A telescopic white cane. They are commonly used by the blind or visually impaired as a mobility tool or as a courtesy to others."
```
Comment (Laniakeaaaa) [inline code/game/objects/items/weapons/cane.dm:82]: And again here by icons

File: code/modules/organs/robolimbs.dm [hunk 6]
```diff
@@ -172,6 +172,22 @@ var/datum/robolimb/basic_robolimb
 	species_cannot_use = list()
 	applies_to_part = list(BP_L_ARM, BP_R_ARM, BP_L_LEG, BP_R_LEG, BP_L_FOOT, BP_R_FOOT, BP_L_HAND, BP_R_HAND)
 
+
+/datum/robolimb/cenilimisybernetics
+	company = "Cenilimi Cybernetics"
+	desc = "This prosthetic is created by a Teshari-owned company, for Teshari."
+	icon = 'icons/mob/human_races/cyberlimbs/cenilimicybernetics/cenilimicybernetics.dmi'
```
Comment (Laniakeaaaa) [inline code/modules/organs/robolimbs.dm:179]: icon = 'icons/uristmob/species, make a resomi/tesh folder, add relevant icon = changes to support it.

File: code/modules/organs/robolimbs.dm [hunk 7]
```diff
@@ -172,6 +172,22 @@ var/datum/robolimb/basic_robolimb
...
+	restricted_to = list(SPECIES_RESOMI)
+	species_cannot_use = list()
+
+/datum/robolimb/unbrandedteshari
+	company = "Unbranded - Teshari"
+	desc = "This prosthetic is small and fit for nonhuman proportions."
+	icon = 'icons/mob/human_races/cyberlimbs/unbranded/unbranded_teshari.dmi'
```
Comment (Laniakeaaaa) [inline code/modules/organs/robolimbs.dm:186]: Add urist path

Comment (Laniakeaaaa) [inline code/modules/organs/robolimbs.dm:186]: Again, etc.

File: code/modules/urist/items/uristweapons.dm [hunk 8]
```diff
@@ -197,3 +197,81 @@ Please keep it tidy, by which I mean put comments describing the item before the
 	throwforce = 20
 	w_class = 4.0
 	slot_flags = SLOT_BELT|SLOT_BACK
+
```
Comment (Laniakeaaaa) [inline code/modules/urist/items/uristweapons.dm:200]: Great work. Make sure you also add the item's name in // (brackets) as a comment version above it. It's more organized and tidy if you do so. E.g, // Crutch  (code would go below this)

**Pull request merged**